adv_project/settings.py

```diff
@@ -26,7 +27,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config('DEBUG', cast=bool)
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=lambda v: [s.strip() for s in v.split(',')])
```

Comment:
I'm looking forward to seeing your deployed project! Great use of the lambda function here to allow for different hosts.